### DISCUSSION TITLE
user-guide/plugins/jasminum

### DISCUSSION THREAD

**Discussion by giscus** [zotero-chinese/wiki#268] [文档评论区]
# user-guide/plugins/jasminum

Zotero 非官方中文维护小组

https://zotero-chinese.com/user-guide/plugins/jasminum

**Comment by NunWang** [comment]
这里update translator已经没有了，看讲解教程

**Reply by northword** [reply]
感谢反馈，我们随后修改，请先参考 https://zotero-chinese.com/user-guide/faqs/update-translators#%E6%AD%A5%E9%AA%A4-3-%E6%9B%B4%E6%96%B0-zotero-connecter-%E7%9A%84%E7%BC%93%E5%AD%98 操作。

**Comment by GANJIUTIAN** [comment]
为什么我的Unofficial Translator Repository中一直显示loading，无法刷新和更新呢？

**Reply by northword** [reply]
请确保插件是最新版本。

**Comment by stan-nicole** [comment]
请问如何“自动设置文献语言”？

**Reply by northword** [reply]
https://zotero-chinese.com/user-guide/plugins/linter

**Comment by zhaozhaocheng147** [comment]
插件是最新版本，请问为什么右键更新知网引用数后，还是无法显示引用数？其他功能都可以使用

**Reply by LLLUPUP** [reply]
同问

**Comment by fuwei529** [comment]
非官方中文翻译器那里一直是loading是怎么回事呢，重新安装插件也不行，插件也是最新版本了，一直都是loading的状态，换了网络也不行

**Reply by AsterismVapour** [reply]
同问

**Comment by tan582** [comment]
茉莉花插件我已经在电脑已经安装好PDFtk了，并且路径选择中我也选择了PDFtk所的保存文件夹中，但是它还是有红色叹号怎么解决呢

**Reply by tan582** [reply]
我是win系统的

**Reply by Soltian** [reply]
一样

**Reply by BelvalLee** [reply]
我也是这样，请问怎么解决呀？

**Comment by amber-gh** [comment]
你好，下载了最新版本的茉莉花插件，插件里显示已启用，但是设置里没有茉莉花，是为什么？

**Reply by northword** [reply]
向茉莉花插件仓库反馈。

**Reply by Soltian** [reply]
我也是这个问题，是不是有bug

**Reply by Soltian** [reply]
设置里面就不显示茉莉花，入口都找不到

**Reply by Soltian** [reply]
解决了，把所有插件都卸载了，重启软件，一个个重新装。装上茉莉花，去设置那看一下有没有。

**Comment by mojie123456** [comment]
提问：为什么按步骤操作完，在Zotero导入的中文论文pdf右键“检索元数据”还是显示“未找到匹配的参考文献”呢？

**Comment by Jeremy-C** [comment]
您好，下载了最新版0.2.9的插件，但选中文献题录条目右键更新知网引用数后，还是无法显示引用数，而且右下角的提示会显示“非核心期刊”（但实际上那个期刊是核心），请问这是什么情况呀？望回复，谢谢！

**Comment by Mn-enge** [comment]
请问在国外是不是无法使用茉莉花抓取知网数据啊，有没有什么代替方法呢

**Reply by lewwellen** [reply]
好像是的，我使用vpn的时候抓取知网就会失败，看到你这条把vpn关了就可以用了。

**Comment by BelvalLee** [comment]
求回复：茉莉花插件我已经在电脑已经安装好PDFtk了，并且路径选择中我也选择了PDFtk所的保存文件夹中，但是它还是有红色叹号，感觉无法解决呢，求解决方法？？？

**Comment by LimeSharon** [comment]
为什么我按照所有流程更新了所有，pdf文献还是无法通过知网抓取信息？

**Reply by Bovit473** [reply]
+1

**Comment by LoongYuan** [comment]
能不能出个mac使用教程啊，我win上安装之后抓取啥的都没问题，但是我mac上试了很多办法 还是抓取失败，用的edge浏览器，更新翻译器，替换源文件，什么办法都试过了，就是不能抓pdf，国外网站的pdf可以抓，国内数据库全不行，万方知网全试过

**Comment by tanlin1867090** [comment]
又无法抓取CNKI学位论文元数据了，服了。

**Comment by scalecontrol** [comment]
插件装好了，PDFtk也安装了，但设置里面没有“非官方维护中文转换器”这个设置窗口是什么问题？多次重新安装zotero和茉莉花插件也没用，都是安装的新版本

**Comment by yzld-sk** [comment]
为什么我的茉莉花不管是在知网直接抓取还是把pdf拖进软件，全都无法抓取元数据呢？已经尝试过在茉莉花和浏览器拓展选项更新转换器。zotero版本是7.0.13，茉莉花版本是1.0.5

**Comment by luosai47** [comment]
zotero6以及放弃维护了

**Comment by 52112566** [comment]
大神，豆瓣的图书评分等元数据能单独抓取到一列么，现在的translator是将所有的评分，定价，评分人数等数据统一抓取在“其他”列中的，不是很方便

**Comment by ttx233** [comment]
为什么抓取文献时抓取不到PDF，显示抓取失败，只能抓取到snapshot

**Comment by yvtzou** [comment]
我已经安装好了pdftk，但是茉莉花里完全不像之前那样有填写pdftk路径的地方了，这是为什么 求助

**Comment by scalecontrol** [comment]
又可能是插件版本的问题，你可以试试在bilibili找相关视频下的茉莉花插件，安装不同版本的插件试一试，不保证一定有用。




ercun
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "zotero-chinese/wiki"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2025年5月25日(星期天) 中午1:36
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [zotero-chinese/wiki] user-guide/plugins/jasminum (Discussion #268)





 
我已经安装好了pdftk，但是茉莉花里完全不像之前那样有填写pdftk路径的地方了，这是为什么 求助

**Comment by DawnRick** [comment]
有大佬知道为啥用爬取谷歌学术的文章到zotero的时候，有的文献能爬取的摘要，有的则不能，但是网页端那些文献都是有摘要的，我想知道为啥，还有就是为啥在zotero查找全文的时候，发现特别慢，几乎不会动，是网速的问题还是什么，我还挂了梯子。。

**Comment by LeoR230212** [comment]
请问一下左侧的目录能固定住吗？每次打开文献都要重新打开一次目录真的好麻烦

**Comment by Fize19** [comment]
对于较旧的知网文献，作者无法自动拆分。老版本的插件支持根据符号拆分作者，希望可以重新添加回来

**Comment by zcybeijing** [comment]
这个插件非常好！能否在插件基础上增加以下功能：1、上传中文专利PDF文件时，从PDF文件中提取专利元数据，并且增加专利条目。2、可以通过输入专利名称/专利号等查询中国专利公告网http://epub.cnipa.gov.cn/，自动添加专利条目和元数据；3、可以通过专利名称/专利号等从中国专利公告网http://epub.cnipa.gov.cn/下载相应的专利文件。

**Comment by shafsysu** [comment]
什么时候更新适合zotero8的啊 跪求

**Comment by zhluo00** [comment]
从本地导入的pdf进行元数据抓取时茉莉花任务窗口的任务一直在转圈圈，无法抓取，怎么办，zotero7，插件是今天最新的了

**Comment by yanruo66** [comment]
请问能不能添加自动化学报，航空学报，宇航学报等检索的支持呀

**Comment by RenXiaokai1999** [comment]
要配置登录页面的时候，点击校外访问需要弹窗，可能是因为zotero的设置问题，无法显示新弹窗，导致配置账号不成功，希望改进

**Comment by 1395639331** [comment]
目录功能已弃用，也就是现在的学位论文都识别不到目录了吗？我的zotero和茉莉花都识别不到目录

**Comment by DawnRick** [comment]
You get the wrong person.
